.codespellignore

```diff
@@ -0,0 +1,3 @@
+doubleclick
+wan
+nwe
```

Comment:
Does it work with
```suggestion
\nWe
```

Reply:
The ignore file is about effectively removing entries from the codespell dictionary, so this change would stop codespell checking for that typo, rather than ignoring that phrase.

The solution would be to use the --ignore-regex option, which will essentially vanish the pattern `\nwe` from the source text that codespell then spellchecks, so it will never find those instances of nwe:
https://github.com/codespell-project/codespell/blob/c807a6cdd1cc88e1a7c75970c855b0b53d4521ac/codespell_lib/_codespell.py#L292-L299

Although I don't think this is in actions-codespell yet, so you'd need to have a codespell config file and use that.
https://github.com/codespell-project/codespell#using-a-config-file
Although it's not yet merged in for reasons I can't immediately remember:
https://github.com/codespell-project/actions-codespell/pull/19

Reply:
Thanks for your input. So, the way to go would be: you merge https://github.com/codespell-project/actions-codespell/pull/19, then we could create a config file which would allow to enable `-ignore-regex`?

Reply:
@peternewman 
Do you have any roadmap for https://github.com/codespell-project/actions-codespell/pull/19?

Reply:
@yubiuser - I'm happy with this PR in it's current state, and then add enhancements later?